docs/toc.md

```diff
@@ -215,6 +215,7 @@
 <!-- The "What's New" section is short, and one level
     deep, so leave it in the main TOC -->
 ## [What's new in C#](csharp/whats-new/index.md)
+### [Configure language version](csharp/whats-new/configure-language-version.md)
```

Comment:
I wouldn't expect to find these topics in a what's new section; they're more reference-like -- possibly as immediate children of the Language Reference topic?